CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file. This library adheres to a versioning policy described in [the README](./README.md#versioning). The public API of this library consists of the functions exported in [h3core.js](./lib/h3core.js).
 
 ## [Unreleased]
+### Added
```

Comment:
Are we adding the PR numbers later or in the PR that adds the Changelog line?

Reply:
Oh, that's a good point, will add.

Reply:
Added.